src/sparseml/export/validators.py

```diff
@@ -17,8 +17,9 @@
 import os.path
 from collections import OrderedDict
 from pathlib import Path
-from typing import Callable, List, Optional, Union
+from typing import Callable, List, Optional
 from typing import OrderedDict as OrderedDictType
+from typing import Union
```

Comment:
lets remove this file diff from the PR

Reply:
This just got in there from running `make style`, its needed for the quality checks to pass